modal/__main__.py

```diff
@@ -1,13 +1,35 @@
 # Copyright Modal Labs 2022
+import sys
+
 from ._traceback import highlight_modal_deprecation_warnings, setup_rich_traceback
 from .cli.entry_point import entrypoint_cli
+from .cli.import_refs import CliUserExecutionError
+from .config import config
 
 
 def main():
     # Setup rich tracebacks, but only on user's end, when using the Modal CLI.
     setup_rich_traceback()
     highlight_modal_deprecation_warnings()
-    entrypoint_cli()
+
+    if config.get("traceback"):
```

Comment:
Could you move this inside the `except Exception as exc` block so you don't need the duplicate code?

Reply:
I could, but then it would add another stack frame to the error

Reply:
Hm I think if you have something like
```python
try:
    entrypoint_cli()
except Exception as exc:
    if show_traceback:
        raise exc
    else:
        print(...)
```
then it would render the as if you'd just called `entrypoint_cli()` with no try/except.

Reply:
Got it!